cursorless-talon/src/actions/generate_snippet.py

```diff
@@ -0,0 +1,71 @@
+import glob
+from pathlib import Path
+
+from talon import Context, Module, actions, settings
+
+from ..targets.target_types import CursorlessExplicitTarget
+
+mod = Module()
+
+ctx = Context()
+ctx.matches = r"""
+tag: user.cursorless_use_community_snippets
+"""
+
+
+@mod.action_class
+class Actions:
+    def private_cursorless_generate_snippet_action(target: CursorlessExplicitTarget):  # pyright: ignore [reportGeneralTypeIssues]
+        """Generate a snippet from the given target"""
+        actions.user.private_cursorless_command_no_wait(
+            {
+                "name": "generateSnippet",
+                "target": target,
```

Comment:
By default don't send dirPath argument. This is for the legacy Cursorless snippet format. The vscode setting for snippet directory will be used.